.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
```

Comment:
On Sun, Mar 15, 2015 at 07:55:49PM -0700, W. Trevor King wrote:
> On Sun, Mar 15, 2015 at 07:53:33PM -0700, Greg Wilson wrote:
> > > @@ -0,0 +1,17 @@
> > > +[project]
> > > +name: lesson-template
> > 
> > Can `update-copyright.py` infer the name of the repo by looking at
> > its Git origin remote instead?
> 
> Hmm, maybe.

I took a VCS-agnostic approach in update-copyright 0.6.2 and just look
at the directory name for the local repository [1,2].

[1]: http://git.tremily.us/?p=update-copyright.git;a=commit;h=88d7ade9e272e079b457c449c931e04994b02429
[2]: http://git.tremily.us/?p=update-copyright.git;a=commit;h=87352a043d30d2812c0db3d65676b9766957773b